config/settings/common.py

```diff
@@ -1,5 +1,8 @@
 """
-:copyright (c) 2014 - 2017, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:copyright (c) 2014 - 2016, The Regents of the University of California,
```

Comment:
I believe all the review issues have been addressed, and the flake8 test is now passing, but would you mind providing some clarification with regards to the line feeds issue so that I can be sure to check for that in any subsequent branches.